sidechain/consensus/aura/src/lib.rs

```diff
@@ -0,0 +1,314 @@
+//! Aura worker for the sidechain.
```

Comment:
Also just a minor issue: Don't we want the license header in all these new files?

Reply:
Very good catch. I have a hard time remembering to put them here.